demo-app/package.json

```diff
@@ -18,7 +18,8 @@
     "grommet-theme-hpe": "^5.0.3",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "react-router-dom": "^6.13.0"
+    "react-router-dom": "^6.13.0",
+    "styled-components": "^5.3.10"
```

Comment:
should this be moved to the root level package.json since it is shared across workspaces? Remove from each local workspace?

Reply:
Done